pkg/storage/store.go

```diff
@@ -3096,6 +3096,7 @@ func (s *Store) processRaftRequest(
 		if req.Message.Type == raftpb.MsgApp {
 			r.setEstimatedCommitIndexLocked(req.Message.Commit)
 		}
+		r.setLastActivityLocked(req.FromReplica.ReplicaID)
```

Comment:
No, that's not possible. The purpose of `Replica.mu.proposalQuotaBaseIndex` is that the index used for quota calculations ratchets up. If a replica comes back on line and has a commit index before the quota base index, it will be ignored for quota purposes until it catches back up.